news/2025/2025-11-20-global-taiko-showdown-concludes.md

```diff
@@ -0,0 +1,282 @@
+
+<div class="news-chat-quote__text-container">
+
+<a class="avatar news-chat-quote__avatar" href="https://osu.ppy.sh/users/3869519" style="background-image: url('/wiki/shared/avatars/Ideal.jpg')"></a>
+
+<p class="news-chat-quote__username"><a class="news-chat-quote__colour-nat" href="https://osu.ppy.sh/users/3869519">Ideal</a></p>
+
+Being a head mappooler not once, but twice, proved itself to be quite a challenge given the very tight deadlines we had to get everything done. Talking to composers about changes to songs to better fit a specific slot in the mappool, working with tens of different mappers and performing difficulty balancing and quality checks on their maps isn’t an easy task, especially for fully custom pools, involving tens of mappers! This was even more evident for me on [MACHKINTV](https://osu.ppy.sh/beatmapsets/2410842#taiko/5234943), for which I was the main coordinator, and it really highlighted how difficult it can be to get everything in order, due to being a 14 mapper megacollab.
+
+That aside, I can say that working with everyone was a pleasure - composers, mappers and designers. Everyone was committed to getting everything done right, and very receptive to whatever changes that were needed to fit everything nicely. This allowed the mappools produced to be of high quality, which definitely showed with how people enjoyed it, players and spectators alike!
+
+</div>
+
+![](/wiki/shared/news/2025-11-20-global-taiko-showdown-concludes/COEGTS_Winner_Banner.png)
+
+<div class="news-chat-quote__text-container">
+
+<a class="avatar news-chat-quote__avatar" href="https://osu.ppy.sh/users/16462012" style="background-image: url('/wiki/shared/avatars/cheesestingy.jpg')"></a>
+
+<p class="news-chat-quote__username"><a class="news-chat-quote__colour-no-group" href="https://osu.ppy.sh/users/16462012">cheesestingy</a></p>
+
+COEGTS was my first (and probably last) LAN tournament. Everything looked like it took a ton of effort, the pools were really fun and well balanced, and I had so much fun playing with friends I’ve spoke to for years. 
+I don’t think I’ve enjoyed myself this much in a week practicing the pool and of course playing on stage, what an unforgettable experience. 
+Thanks to everyone that helped make this come true, from all people involved in GTS work to staff members at COE, and my family for being there with me.
+It’s been a honor.
+
+</div>
+
+Once again, thank you all for working with us for so long! This article officially marks the end of GTS' life as an osu! tournament series. It has been a very bumpy road of 7 years of activity, but we still appreciate it regardless. You may see maps for past GTS original songs get ranked in the future, but this will be the extent of our activity in this scene from now on.
+We will stay active as a music label under the name **Gratia Productions**, which is not planned to be active until mid-2026 minimum regardless. 
+See you around!
```

Comment:
```suggestion
Once again, thank you all for working with us for so long! This article officially marks the end of GTS' life as an osu! tournament series. It has been a very bumpy road of 7 years of activity, but we still appreciate it regardless. You may see maps for past GTS original songs get ranked in the future, but this will be the extent of our activity in this scene from now on.

We will stay active as a music label under the name **Gratia Productions**, which in any case is not planned to be active until mid-2026 at minimum.

See you around!
```